rllib/examples/rl_modules/autoregressive_actions_rl_module.py

```diff
@@ -100,7 +102,7 @@
     # exceeds 150 in evaluation.
     stop = {
         f"{NUM_ENV_STEPS_SAMPLED_LIFETIME}": 100000,
-        f"{EVALUATION_RESULTS}/{ENV_RUNNER_RESULTS}/{EPISODE_RETURN_MEAN}": 150.0,
+        f"{EVALUATION_RESULTS}/{ENV_RUNNER_RESULTS}/{EPISODE_RETURN_MEAN}": -0.012,
```

Comment:
nice!

Reply:
Where does it roughly start?

Reply:
It roughly starts at around `-0.55` - `-0.6`

Reply:
![ray_tune_evaluation_env_runners_agent_episode_returns_mean_default_agent](https://github.com/user-attachments/assets/777cf02c-a8e3-441f-8d20-282d3ec5727d)

Reply:
Niiice!!